psi4/src/psi4/libmints/deriv.cc

```diff
@@ -618,13 +746,13 @@ SharedMatrix Deriv::compute(DerivCalcType deriv_calc_type) {
 
     // Print things out, after making sure that each component is properly symmetrized
     enuc->symmetrize_gradient(molecule_);
-    opdm_contr_->symmetrize_gradient(molecule_);
-    x_contr_->symmetrize_gradient(molecule_);
+    opdm_contr->symmetrize_gradient(molecule_);
+    x_contr->symmetrize_gradient(molecule_);
     tpdm_contr_->symmetrize_gradient(molecule_);
 
     enuc->print_atom_vector();
-    opdm_contr_->print_atom_vector();
-    x_contr_->print_atom_vector();
+    opdm_contr->print_atom_vector();
+    x_contr->print_atom_vector();
     tpdm_contr_->print_atom_vector();
```

Comment:
Same here.

Reply:
New behavior: For conventional and DF integrals, at print level 1, cdsalc components are not printed, and neither are gradient components. At print level 2, gradient components are printed, _but_ in the DF case, all TEI terms are folded together. At print level 3, cdsalc components are printed, and in the DF case, TEI terms are not folded together.